docs/UserGuide.md

```diff
@@ -110,10 +109,11 @@ Examples:
 
 Edits the existing resident record at a specified index.
 
-Format: `redit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [r/ROOM]`
+Format: `redit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL]`
 * `INDEX` refers to the index number shown in the displayed resident list. `INDEX` **must be a positive integer: 1, 2, 3, …**.
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* `redit` will be blocked if the resident has an active room allocation. `dealloc` before making further edits. 
```

Comment:
Just a syntactic/clarity thing: Perhaps update the sentence to say "Run `dealloc` on the resident you are editing first." and then provide an example command

Reply:
Ok. Added the example command to the userGuide.